components/style/properties/data.py

```diff
@@ -14,6 +14,14 @@
 ALL_SIZES = [(size, False) for size in PHYSICAL_SIZES] + [(size, True) for size in LOGICAL_SIZES]
 
 
+def maybe_moz_logical_alias(product, side, prop):
```

Comment:
Please leave a comment here that `inline-start` and `-end` props can omit the `inline`.